modules/exploits/windows/local/service_permissions.rb

```diff
@@ -268,6 +274,12 @@ def exploit
     tmpdir = dir_env['TEMP']
     tempexe = "#{tmpdir}\\#{tempexe_name}"
 
+    if target.name == 'Exploit CVE-2025-21293'
```

Comment:
Maybe worth trying to check to see if UAC is enabled?
I feel like there must be a lib/msf/post/windows method to check, but the best I could find is 
https://github.com/rapid7/metasploit-framework/blob/74d828c73de41288699272959553c5ddd35db555/scripts/meterpreter/getcountermeasure.rb#L339